### DISCUSSION TITLE
Setup guide for creating LoRA

### DISCUSSION THREAD

**Discussion by elistys** [viking1304/a1111-setup#8] [General]
Hello
Thank you for the great setup guide.
I'm asking because I think you're a specialist who is familiar with stable diffusion on Mac.

As a further development, is it possible to provide a setup guide that can reliably create LoRA on Mac(Apple Silicon)+A1111?

There are reports in [this thread](https://github.com/bmaltais/kohya_ss/issues/1248) that training has been started, but I have never seen a report of a LoRA that actually works on the webUI.

I also tried copying the success story from that thread, but I couldn't create it. I'm skeptical that LoRA can be created on Mac+A1111.
By the way, I was able to create LoRA with an app called Draw Things.

**Comment by viking1304** [comment]
Sorry, I have been very busy lately and forgot to answer.

In theory, I can create an installer, but I have never used kohya_ss.

That guide looks fine, so it should work without any issues. There is no reason for LORA to behave differently depending on the platform used for training. The only reason I could think of is that LORA is not converted to safetensors. It would be best to convert it to safetensors. Disabling checks for unsafe files will probably work, but I do not recommend that.

btw. please run this command on your Mac and post the result:

```
❯ sh a1111-setup.sh -i
```

**Reply by elistys** [reply]
Thank you for your reply.

In my case, I couldn't create a LORA in the first place. I haven't seen any reports of people being able to create a working LoRA in the guide you linked to.

I thought it would be very difficult,or impossible, to create a LoRA using the web UI on a Mac.

I'm not a programmer, so I don't understand what that command does. I looked it up but couldn't figure it out, so I didn't run it. Sorry.
